### ISSUE TITLE
Submit for inclusion in slime distribution?, + feature

### ISSUE THREAD

**Issue by alanruttenberg** [anwyn/slime-company#9]
Hi,
I didn't know about this until recently and thought it might get more use if you included it in the slime distribution at https://github.com/slime/slime

Also, I added an extension that uses slime-company but completes over symbols in all packages. It's written as a separate contrib right now but thought I would see if you want to integrate something like this directly into slime-company. 

Selfishly, if I give my contrib to the slime project it would be easier if yours was there as well, since I build on it.

Source at https://github.com/alanruttenberg/slime/blob/slime-company/contrib/slime-allsymbols-company.el and corresponding .lisp

Suggestions welcome, of course.

**Comment by anwyn** [comment]
Hey Alan,

I have never really considered slime-company for inclusion into Slime. Back when I wrote it, Slime maintainer Helmut would even keep the fancy inspector out of the slime base install, so I reckoned something even more 'fancier' like slime-company, that, on top, depends on another external package would never make it into Slime. This might have changed now, with different maintainers, but today we have MELPA, which seem to work nicely, pulls in dependencies and lets you have your own license and release schedule. 
So I would recommend: Make a MELPA package from your contrib and be done with it.
